README.md

```diff
@@ -0,0 +1,29 @@
+# Astroneer Modloader
+
+A modloader for Astroneer, rewritten in Rust.
+
+## Installation
+
+### Windows
+
+Download the modloader (`astro_modloader.exe`) from the [releases
+page](https://github.com/AstroTechies/astro_modloader/releases/latest), below the changelog.
+
+### Linux
+
+Pre-built binaries are not yet dsitributed for Linux. To build the modloader yourself on Linux:
+
+- Use your distribution's package manager to install `rustup`, `git`, and `build-essential`,
```

Comment:
maybe make a note that `build-essential` is distro specific?

Reply:
Done! I also added a note that `build-essential` might be installed by `rustup`.